Cargo.toml

```diff
@@ -1,73 +1,73 @@
 [workspace]
 members = [
-    "crypto/aead",
```

Comment:
:grimacing: Ooops. I have a toml LSP now, and it looks like it likes auto-formatting...

Reply:
Should we just keep it? :D

`taplo` seems like somewhat standard LSP for toml, and it's in Rust ...

Reply:
How do your text editors treat these 2-space indent sizes?